src/NzbDrone.Core/IndexerSearch/NzbSearchService.cs

```diff
@@ -257,6 +260,13 @@ private List<DownloadDecision> Dispatch(Func<IIndexer, IEnumerable<ReleaseInfo>>
 
             foreach (var indexer in indexers)
             {
+                var indexerStatus = _indexerStatusService.GetIndexerStatus(indexer.Definition.Id);
+                if (indexerStatus != null && indexerStatus.DisabledTill.HasValue && indexerStatus.DisabledTill.Value > DateTime.UtcNow)
+                {
+                    _logger.Debug("Temporarily ignoring indexer {0} till {1} due to recent failures.", indexer.Definition.Name, indexerStatus.DisabledTill.Value);
```

Comment:
They already have warnings of the earlier errors. If you do this as Info you get alot of entries saying the same thing over and over again.